osidb/models.py

```diff
@@ -2988,6 +2996,14 @@ class Meta:
             GinIndex(fields=["acl_read"]),
         ]
 
+    @property
+    def time(self):
+        """
+        Return the time of the BZ comment if sync is enabled, or the
+        record's creation datetime otherwise.
+        """
+        return self.meta_attr.get("time", self.created_dt)
```

Comment:
I am thinking why is this necessary. Wouldn't it be better to just set `created_dt` to the Bugzilla `time` value in the `FlawConvertor` if fetched from Bugzilla and just leave it to be automatically set by the `TrackingMixin` otherwise? When/why should they be different?

Reply:
I did that because `meta_attr.time` is what OSIM checks to make the transition more clear, but thinking it twice, I think you're right since we shouldn't bend OSIDB to adhere to OSIM standards, but the other way around. So I will definitely change that.